show/main.py

```diff
@@ -1,20 +1,20 @@
 #! /usr/bin/python -u
 
 import json
-import netaddr
+#import netaddr
 import os
 import subprocess
 import sys
 
 import click
 from natsort import natsorted
 import netifaces
-from pkg_resources import parse_version
+#from pkg_resources import parse_version
```

Comment:
Why comment out?

Reply:
This module import is taking more time(~300ms) on arm cpu. As this is required only for "show ntp" cmd, I have added it inside ntp command. see below